components/utils/calculation

```diff
@@ -0,0 +1,8 @@
+export default function calculation(value, expense, coin) {
+  const priceCrypto = coin.current_price;
+  let result = 0;
+  if (expense) {
+    result = ((value * priceCrypto) / expense).toFixed(0);
+  }
+  return result;
+}
```

Comment:
I would move this function outside of the components folder.
Create a folder `utils` or `lib` on the same level as your components and pages.
Make sure to give your file an extension (`.js`).